selfdrive/car/volkswagen/interface.py

```diff
@@ -13,6 +13,13 @@ def __init__(self, CP, CarController, CarState):
     self.displayMetricUnitsPrev = None
     self.buttonStatesPrev = BUTTON_STATES.copy()
 
+    if CP.networkLocation == NetworkLocation.fwdCamera:
+      CANBUS.ext = CANBUS.pt
```

Comment:
not a fan of this. think this would be cleaner
```suggestion
      self.ext_bus = CANBUS.pt
```

Reply:
Done.